Source/WebKit/WebProcess/GPU/media/RemoteVideoCodecFactory.cpp

```diff
@@ -177,8 +177,8 @@ void RemoteVideoCodecFactory::createEncoder(const String& codec, const WebCore::
     std::map<std::string, std::string> parameters;
     if (type == VideoCodecType::H264) {
         if (auto position = codec.find('.');position != notFound && position != codec.length()) {
-            auto profileLevelId = StringView(codec).substring(position + 1);
-            parameters["profile-level-id"] = std::string(reinterpret_cast<const char*>(profileLevelId.characters8()), profileLevelId.length());
+            auto profileLevelId = spanReinterpretCast<const char>(StringView(codec).substring(position + 1).span8());
```

Comment:
I don’t think `StringView` needs to be involved here. Instead `codec.span8().subspan(position + 1)`